Neos.Cache/Tests/Functional/Backend/RedisBackendTest.php

```diff
@@ -54,9 +58,9 @@ public function setUp()
             $this->markTestSkipped('redis server not reachable');
         }
         $this->backend = new RedisBackend(
-            new ApplicationContext('Testing'), array('hostname' => '127.0.0.1', 'database' => 0)
+            new EnvironmentConfiguration('Redis a wonderful color Testing', '/some/path', PHP_MAXPATHLEN), ['hostname' => '127.0.0.1', 'database' => 0]
         );
-        $this->cache = $this->createMock(\TYPO3\Flow\Cache\Frontend\FrontendInterface::class);
+        $this->cache = $this->getMock(FrontendInterface::class);
```

Comment:
No, it must be `createMock()`, `getMock()` is deprecated and will trigger warnings.